pshtt/pshtt.py

```diff
@@ -226,6 +245,12 @@ def basic_check(endpoint):
         except requests.exceptions.RequestException:
             # Swallow connection errors, but we won't be saving redirect info.
             pass
+        except Exception as err:
+            endpoint.unknonwnerror = True
```

Comment:
(Also, I'm fixing it myself, no need to add a commit. Am working locally on prepping this for merge.)